### PR TITLE
[DOCS-7126] Alfresco Enterprise Viewer 3.5 docs improvements

### PR DESCRIPTION
**Pull request by anxumalo**
This PR replaces the updates started in `enterprise-viewer-post-3.5` branch.

### CODE REVIEW

**Review by katiebrill — DISMISSED** [review]
Looks good to me - thank you!

File: enterprise-viewer/latest/config/files.md [hunk 1]
```diff
@@ -1,610 +1,647 @@
...
-Whether or not the user wants a modal to appear asking them to confirm that they want to save page modifications when in edit mode.
+Whether or not the user wants a modal dialog box to appear asking them to confirm that they want to save page modifications when in edit mode.
 
 Default value: `false`
 
-#### controlledPrint
+### controlledPrint
 
-Whether or not we want a controlled print to be enabled. Controlled print only allows certain people to print a document, and controls the number of times it can be printed.
+Whether or not controlled printing ia enabled. Controlled print only allows certain people to print a document, and controls the number of times it can be printed.
```
Comment (Amy-Hyland) [inline enterprise-viewer/latest/config/files.md:455]: I think "ia" was meant to be "is"?

File: enterprise-viewer/latest/install/index.md
Comment (Amy-Hyland) [inline enterprise-viewer/latest/install/index.md:1]: I think Apache should be capitalized, and Nginx may also need to be referenced as NGINX.

File: enterprise-viewer/latest/install/index.md [hunk 2]
```diff
@@ -709,3 +607,116 @@ This section walks through how to install the web applications on Alfresco Tomca
...
+
+    ```bash
+    java\{javaVersion}\bin\java -jar {ALFRESCO_HOME}\bin\alfresco-mmt.jar install {ALFRESCO_HOME}\amps_share\oa-share-webpreview.amp tomcat\webapps\share.war -force 
+    ```
+
+5. Delete current Share deployed WAR files
+
+   Navigate to the `ALFRESCO_HOME/tomcat/webapps` directory and delete the `share` folder (if it exists)
+
+6. Start alfresco
```
Comment (Amy-Hyland) [inline enterprise-viewer/latest/install/index.md:716]: Capitalize Alfresco or specifically name the Alfresco application being used.

File: enterprise-viewer/latest/using/index.md [hunk 3]
```diff
@@ -1517,22 +1494,19 @@ Clicking this button will allow the user to split the document into different se
 
 ![Img Txt]({% link enterprise-viewer/images/aev-edit-pages-interface-toolbar14.png %})
 
-Sectioning is an efficient way to reorder large documents. A blue notification will appear in the bottom right corner
-where the user can name the section and select the range of pages for the section. The user must then select the create
-section button. The new section will appear in the left sidebar. For more information on how to section a document,
-see this [section](#section-docs).
+Sectioning is an efficient way to reorder large documents. A blue notification will appear in the bottom right corner where the user can name the section and select the range of pages for the section. The user must then select the create section button. The new section will appear in the left sidebar.
```
Comment (Amy-Hyland) [inline enterprise-viewer/latest/using/index.md:1497]: Is "create section" an actual button? If it is, I suggest using the bold font face to maintain consistent formatting.

**Review by Amy-Hyland — CHANGES_REQUESTED** [review]
I've left a few comments and added a commit. Otherwise everything builds and is ok.

File: enterprise-viewer/latest/using/index.md (hunk 3 above)
Comment (anxumalo) [reply enterprise-viewer/latest/using/index.md:1497]: Good spot - I missed this one! It's a UI element and worth highlighting as it's a link inside a notification: see https://docs.alfresco.com/enterprise-viewer/latest/using/#creating-a-section.

**Review by Amy-Hyland — APPROVED** [review]
LGTM

**Pull request merged**